test/framework/src/main/java/org/opensearch/bootstrap/BootstrapForTesting.java

```diff
@@ -171,8 +171,8 @@ public boolean implies(ProtectionDomain domain, Permission permission) {
                 });
                 // Create access control context for mocking
                 PriviledgedMockMaker.createAccessControlContext();
-                System.setSecurityManager(SecureSM.createTestSecureSM(getTrustedHosts()));
-                Security.selfTest();
+//                System.setSecurityManager(SecureSM.createTestSecureSM(getTrustedHosts()));
+//                Security.selfTest();
```

Comment:
If we don't need these lines, let's remove them.

Reply:
yes, it was pushed by mistake, corrected this.